Gemfile.lock

```diff
@@ -588,6 +590,10 @@ GEM
     rspec-expectations (3.13.0)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.13.0)
+    rspec-json_matchers (0.1.0)
```

Comment:
Is it feasible to avoid pulling in a new dependency for the current usage?

Reply:
Sure, done in 97d71644c4b0a9f18fd88d98967c8922f4d09d19